modules/installation-vsphere-infrastructure.adoc

```diff
@@ -52,34 +52,34 @@ You must install the {product-title} cluster on a VMware vSphere version 7 insta
 [cols=2, options="header"]
 |===
 |Virtual environment product |Required version
-|VM hardware version | 15 or later
+|VMware virtual hardware version | 15 or later
```

Comment:
VMware virtual hardware version --> VMware virtual hardware

(you don't need version in the column name since the table header already says "Required version" in the header.